documents/knowledge/elements.yaml

```diff
@@ -4349,7 +4349,7 @@
     lst: Diagonals of a matrix, starting with the main diagonal.
   vectorise: false
   tests:
-    - "[[[1,2,3],[4,5,6],[7,8,9]]] : [[1,5,9],[2,6],[3],[4,8],[7]]"
```

Comment:
Is this just a minor bugfix?

Reply:
Nah, it's just that my implementation returns them in a different order. We currently have no way for our tests to check things without the order, I should make an issue for that.